config/locales/views/en.yml

```diff
@@ -70,6 +70,9 @@ en:
     proxy_notice:
       text_html: "Notice: (1) You are using a proxy site that is not affiliated with AO3. (2) The entity that set up the proxy site can see whatever data you submit to the AO3 through the proxy site, including your IP address and the content of comments."
       button: "Dismiss Notice"
+    home:
```

Comment:
Even though this is technically used by a file in `layouts`, I think we can move it out from under there and treat it like [the subtitle for the troubleshoot controller's show page](https://github.com/otwcode/otwarchive/blob/a3a88a4ed830db48a5fe35f590318792ac7805b8/app/controllers/troubleshooting_controller.rb#L10-L14).

Reply:
I made the suggested change and it works fine.

Nitpicking my own code:
It looks odd to have `@page_subtitle = t(".page_title")`, where key is named "title", but it only has one use, which is to be a subtitle.  The example you gave does the same thing though.

Reply:
Yeah, we've got  some renaming to do around `@page_title` and `@page_subtitle`, but for now, this is fine!